packages/squiggle-lang/__tests__/Symbolic_test.res

```diff
@@ -0,0 +1,30 @@
+open Jest
+open Expect
+open Js.Array
+open SymbolicDist
+
+let makeTest = (~only=false, str, item1, item2) =>
+  only
+    ? Only.test(str, () => expect(item1) -> toEqual(item2))
+    : test(str, () => expect(item1) -> toEqual(item2))
+
+let normalParams1: SymbolicDistTypes.normal = {mean: 5.0, stdev: 2.0}
+// let normalParams2: SymbolicDistTypes.normal = {mean: 10.0, stdev: 2.0}
+let normalParams3: SymbolicDistTypes.normal = {mean: 20.0, stdev: 2.0}
+let range20 = [0.0,1.0,2.0,3.0,4.0,5.0,6.0,7.0,8.0,9.0,10.0,11.0,12.0,13.0,14.0,15.0,16.0,17.0,18.0,19.0,20.0]
```

Comment:
We should probably have a function that does this (I think I did something similar in my other story, and also would want that to be improved there)

Reply:
This util would be in `Sparklines` and not in `E`?

Reply:
If sparklines had a better interface like what I mention, then stuff would be in sparklines. 

However, we should probably just have one function create a range of integers (I'm pretty sure one is already in the standard libraries; if it doesn't creating it seems good). This function would go in E. (Everything generic to basic types goes there)

Reply:
> https://rescript-lang.org/docs/manual/latest/api/belt/array

util for this with float casting is in `E.A`